.gitignore

```diff
@@ -1,2 +1,2 @@
 /vendor/
-.idea
```

Comment:
Note: removing this because it should be ignored globally on dev workstation, it's not project-specific.